extensions/eda/plugins/event_filter/insert_hosts_to_meta.py

```diff
@@ -14,6 +14,8 @@
                     string but contains multiple hosts, use this parameter to
                     delimits the hosts. Treat the vale as a single host if the
                     parameter is not present.
+    raise_error:    Whether raise PathNotExistError if host_path does not
+                     exist in the event. Default to false
```

Comment:
@bzwei I think we should mention that while developing the rulebook its recommended that you set the raise_error to True and once you are content that it works you can set it to false.